vm/src/vm.rs

```diff
@@ -583,4 +588,12 @@ impl GlobalVmState {
     pub fn get_env<'b>(&'b self) -> RwLockReadGuard<'b, VmEnv> {
         self.env.read().unwrap()
     }
+
+    pub fn get_debug_level<'b>(&'b self) -> RwLockReadGuard<'b, DebugLevel> {
```

Comment:
I think returning just `DebugLevel` here would be fine, or do you think there may be a reason not to clone/copy it?

Reply:
That's a good point but I think i'd still prefer copying. I may end up refactoring here to put more values under the same `RwLock` so being able to return the lock around just `DebugLevel` would need to change then.